lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -24,6 +25,8 @@ def setUp(self):
         super().setUp()
         self.url = reverse('course-home-outline-tab', args=[self.course.id])
 
+    @COURSE_HOME_MICROFRONTEND.override(active=True)
+    @COURSE_HOME_MICROFRONTEND_OUTLINE_TAB.override(active=True)
```

Comment:
I believe you can do this at the class level too? Might be easier (and then override to False if you need to for like, an inactive test)

Reply:
There was some wonky behavior here when I attempted this, so we created this ticket: https://openedx.atlassian.net/browse/AA-268